services/QuillLMS/client/app/bundles/Diagnostic/libs/grading/rematching.ts

```diff
@@ -75,14 +86,14 @@ export function rematchAll(mode: string, question: Question, questionID: string,
   });
 }
 
-export function rematchOne(response: string, mode: string, question: Question, questionID: string, callback:Function) {
+export function rematchOne(response: Response, mode: string, question: Question, questionID: string, callback:Function) {
   const matcher = getMatcher(mode);
   getGradedResponses(questionID).then((data) => {
     question.key = questionID
     const matcherFields = getMatcherFields(mode, question, formatGradedResponses(data));
     const promise = rematchResponse(matcher, matcherFields, response);
     if (promise) {
-      promise.then(() => { callback(); });
+      promise.then((result) => { callback(result); });
```

Comment:
It doesn't look like we're doing something with the `result` in `searchResponses`, am I missing something?